src/Libraries/PythonNodeModels/PythonNode.cs

```diff
@@ -65,12 +66,12 @@ public PythonEngineVersion Engine
         /// Available Python engines.
         /// </summary>
         [Obsolete(@"This method will be removed in future versions of Dynamo.
-        Please use PythonEngineSelector.Instance.AvailableEngines instead")]
+        Please use PythonEngineManager.Instance.AvailableEngines instead")]
         public static ObservableCollection<PythonEngineVersion> AvailableEngines
         {
             get
             {
-                return PythonEngineSelector.Instance.AvailableEngines;
+                return new ObservableCollection<PythonEngineVersion>(PythonEngineManager.Instance.AvailableEngines.Select(x=> x.Version));
```

Comment:
returning a new Observable on getter will loose old subscribers...but since this method is obsolete...it should not matter that much